create_dmd_release/create_dmd_release.d

```diff
@@ -372,7 +372,10 @@ void buildAll(Bits bits, string branch, bool dmdOnly=false)
     auto hostDMDEnv = " HOST_DC="~hostDMD~" HOST_DMD="~hostDMD;
     auto isRelease = " ENABLE_RELEASE=1";
     //Enable lto for everything.
-    auto ltoOption = " ENABLE_LTO=1";
+    version (FreeBSD)
+        auto ltoOption = " ENABLE_LTO=0";
```

Comment:
comment is outdated

Reply:
Added the rationale why LTO is disabled for FreeBSD